inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -332,24 +334,31 @@ void MKLDNNGraph::InitGraph() {
 
     SortTopologically();
     InitNodes();
+
     optimizer.ApplyCommonGraphOptimizations(*this);
     SortTopologically();
 
     InitDescriptors();
 
-    for (auto &node : graphNodes) {
-        node->initOptimalPrimitiveDescriptor();
-    }
+    InitOptimalPrimitiveDescriptors();
+
     InitEdges();
 
     optimizer.ApplyImplSpecificGraphOptimizations(*this);
-
     SortTopologically();
 
     Allocate();
 
     CreatePrimitives();
 
+    InitGraphRemainingActions();
```

Comment:
please rename or just keep it unmodified.

Reply:
fixed